cnd/src/db/models.rs

```diff
@@ -6,57 +6,214 @@ use diesel::{
     sql_types::Text,
     Insertable, Queryable, *,
 };
-use std::{io::Write, str::FromStr, string::ToString};
+use failure::_core::{
```

Comment:
:mega: `Debug`, `Display` and `Deref` should come from the standard library. `failure` is just re-exporting them here.

Reply:
Also, they are always in scope, no need to import them at all.